configure.ac

```diff
@@ -1305,6 +1305,9 @@ AS_IF([$has_c99_float_ops],
          C99 float ops unavailable, replacements enabled
          (ancient Visual Studio)]))])])])
 
+## getentropy
+AC_CHECK_FUNC([getentropy], [AC_DEFINE([HAS_GETENTROPY])])
```

Comment:
I'd rather use `AC_CHECK_DECL` to check for a visible declaration in `<unistd.h>` or in the Apple-specific header.  Under Linux for instance, `getentropy` is not declared if the user compiles in strict ISO C mode or in POSIX_C_SOURCE mode.